launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light.launch.xml

```diff
@@ -6,6 +6,7 @@
   <arg name="input/camera_info" default="/sensing/camera/traffic_light/camera_info" description="camera info topic name for traffic light"/>
   <arg name="fine_detector_label_path" default="$(find-pkg-share traffic_light_ssd_fine_detector)/data/voc_labels_tl.txt" description="fine detector label path"/>
   <arg name="fine_detector_model_path" default="$(find-pkg-share traffic_light_ssd_fine_detector)/data/mb2-ssd-lite-tlr.onnx" description="fine detector onnx model path"/>
+  <arg name="fine_detector_is_box_first" default="false"/>
```

Comment:
I would like to change the name of a variable.
Difficult to understand the meaning of variable.

Reply:
Do you have any ideas about that?
My ideas are following...
- `fine_detector_is_box_head_first`
- `fine_detector_is_box_score_order`
- `fine_detector_is_box_score_head_order`

Reply:
discussion memo:
```
<arg name="dnn_header_type" default="pytorch" docs="pytorch or mmdetection" />
```

Reply:
@yukkysaito Add a parameter named `dnn_header_type` by https://github.com/autowarefoundation/autoware.universe/pull/3485/commits/bef8fe324f8e19cb001dba37d88503e9fab39856